infra/relayer/src/relayer.js

```diff
@@ -48,28 +48,57 @@ class Relayer {
    */
   constructor(networkId) {
     this.networkId = networkId
+    let privateKey
 
     // Load contract addresses based on network id.
     switch (networkId) {
       // Mainnet
       case 1:
         this.addresses = require('@origin/contracts/build/contracts_mainnet.json')
+        this.web3 = new Web3(
+          'https://mainnet.infura.io/v3/98df57f0748e455e871c48b96f2095b2'
+        )
+        privateKey = process.env.FORWARDER_PRIVATE_KEY_1
```

Comment:
Should this just be FORWARDER_PK without network and we can set different value for network 1,4,2222 in EnvKey ?